src/main/scala/de/zalando/aruha/nakadi/OperationsScala.scala

```diff
@@ -1,6 +0,0 @@
-package de.zalando.aruha.nakadi
```

Comment:
It's dead code. I'd prefer to delete it. Our door to Scala world is always open :)